recipes/poly2tri/all/CMakeLists.txt

```diff
@@ -1,34 +1,29 @@
 cmake_minimum_required(VERSION 3.4)
-project(poly2tri)
-
-include(conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
+project(poly2tri LANGUAGES CXX)
 
 include(GNUInstallDirs)
 
-set(SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder/poly2tri)
-
-add_library(${PROJECT_NAME}
-  ${SRC_DIR}/common/shapes.cc
-  ${SRC_DIR}/sweep/advancing_front.cc
-  ${SRC_DIR}/sweep/cdt.cc
-  ${SRC_DIR}/sweep/sweep.cc
-  ${SRC_DIR}/sweep/sweep_context.cc
+add_library(poly2tri
+    ${POLY2TRI_SRC_DIR}/common/shapes.cc
+    ${POLY2TRI_SRC_DIR}/sweep/advancing_front.cc
+    ${POLY2TRI_SRC_DIR}/sweep/cdt.cc
+    ${POLY2TRI_SRC_DIR}/sweep/sweep.cc
+    ${POLY2TRI_SRC_DIR}/sweep/sweep_context.cc
```

Comment:
I was not paying attention to this, but according to CMake's doc, it should be `SOURCE` instead of `SRC`: https://cmake.org/cmake/help/v3.24/variable/PROJECT-NAME_SOURCE_DIR.html. It is clear that this one works, I wonder if it is an error on CMake docs (I'm not testing it now)

Reply:
This variable `POLY2TRI_SRC_DIR` is unrelated. It's injected externally to honor the layout from conanfile.